postgresql/client.go

```diff
@@ -331,6 +331,8 @@ func (c *Client) Read(req *prompb.ReadRequest) (*prompb.ReadResponse, error) {
 				return nil, err
 			}
 
+			log.Debug("time", time, "labels", labels, "name", name, "value", value)
```

Comment:
@niksajakovljevic I've added the conditional.  

re:
> Also it would be more valuable to actually log the Prometheus protobuf messages that are being sent out.

I'm not really sure which one is the prometheus model here.  Do you mean log the `promdb.Sample` object before adding it to `ts.Samples` array?

Reply:
Yeah, but since `promdb.Sample` does not contain labels maybe logging `prompb.TimeSeries` object after it has been appended to the response:
`resp.Results[0].Timeseries = append(resp.Results[0].Timeseries, ts)`?

Reply:
https://github.com/timescale/prometheus-postgresql-adapter/pull/34/commits/c48b3bf04f6fec58d098745ce787201044cd199b

something like that?

Reply:
Was thinking of `log.debug("ts", ts.String())` after https://github.com/timescale/prometheus-postgresql-adapter/blob/master/postgresql/client.go#L379

Reply:
Just realized I didn't push the latest change.  I did the same thing as you suggested: https://github.com/timescale/prometheus-postgresql-adapter/pull/34/commits/0a0d2ed12b3e267974657d86bd401996e078c295